### ISSUE TITLE
Cant import data

### ISSUE THREAD

**Issue by typicalsmc** [YiC200333/XConomy#60]
**Version of XConomy**  
XConomy 2.18.4

**Server version**  
Paper 1.18.1

**Storage-type**  
MySQL

**Describe the issues**  
Issues
Import data from Essentials doesnt work

---

**(option)XConomy's config.yml**

**(option)Screenshots**

**Comment by YiC200333** [comment]
can u find import data in this folder(XConomy/importdata/data.yml)
after your importing, the data will import to xconomy's database when player join the game.

**Comment by typicalsmc** [comment]
yes, but it isnt full, there few players with millions money, which doesnt have their name and money in the data.yml

**Comment by YiC200333** [comment]
The current version of import data is to obtain the full server players list through `Bukkit.getServer().getOfflinePlayers()`
This method may lose some players, but I don't have a better solution at present.
